### PR TITLE
Update documentation (User Guide, DeleteCommand, UI)

### PR DESCRIPTION
**Pull request by jordanyoong**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -42,76 +40,80 @@ Common Cents is your convenient at-hand expense-tracking tool, meant for anyone
 
 <div markdown="block" class="alert alert-info">
 
-**:information_source: Notes about the command format:**<br>
+**Notes about the command format:**<br>
 
 * Words in UPPERCASE are the parameters to be supplied by the user.<br>
-  e.g. in `add n/EXPENSE_NAME`, `EXPENSE_NAME` is a parameter which can be used as `add n/Buying cups`
+  e.g. in `add C/CATEGORY...`, `CATEGORY` is a parameter which can be used as `add c/REVENUE...`
```
Comment (Ziyang-98) [inline docs/UserGuide.md]: the "C" in "C/CATEGORY" should be lowercase

File: docs/UserGuide.md [hunk 2]
```diff
@@ -150,9 +139,9 @@ Format: `exit`
...
 **Help** | `help`
-**Add** | `add c/CATEGORY  n/EXPENSE_NAME a/EXPENSE_AMT`
-**Remove** | `remove ENTRY_NUM`<br>`remove n/EXPENSE_NAME`
-**Edit** | `edit ENTRY_NUM n/EXPENSE_NAME a/MONEY_AMT`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`
-**List** | `list CATEGORY`
+**Add** | `add c/CATEGORY d/DESCRIPTION a/AMOUNT [t/TAG]`
+**Delete** | `delete ENTRY_INDEX c/CATEGORY`
 **Exit** | `exit`
+
```
Comment (Ziyang-98) [inline docs/UserGuide.md:145]: This is still Work In Progress right? Since by wednesday there will be new commands

**Review by Ziyang-98 — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (jordanyoong) [reply docs/UserGuide.md:145]: Yup will add once they're implemented! Some are already inside but commented out

Comment (Ziyang-98) [reply docs/UserGuide.md:145]: oh ok roger

**Review by Ziyang-98 — APPROVED** [review]
LGTM!

**Pull request merged**